### DISCUSSION TITLE
Google / AWS KMS support

### DISCUSSION THREAD

**Discussion by johntdyer** [envio-cli/envio#50] [Ideas]
Google KMS integration like done in [SOPS](https://github.com/mozilla/sops)

**Comment by humblepenguinn** [comment]
Hi @johntdyer thank you for using `envio` and for your idea!

I haven't really used Google KMS before so I am going to have to take a look at that and how `SOPS` integrates it within their tool. It sounds like a great idea and is something for sure I will put into `envio`'s future enhancements!

Alongside Google KMS do you think Amazon KMS is something users would find useful?
